src/BenchmarkDotNet/Toolchains/IToolchain.cs

```diff
@@ -14,5 +17,6 @@ public interface IToolchain
         bool IsInProcess { get; }
 
         bool IsSupported(BenchmarkCase benchmarkCase, ILogger logger, IResolver resolver);
```

Comment:
Is `IsSupported` still needed? We should consider removing it.

Reply:
Yes, still used